.circleci/config.yml

```diff
@@ -1,8 +1,10 @@
-version: 2
+version: 2.1
+orbs:
+  codecov: codecov/codecov@1.0.5
 jobs:
   build:
     docker:
-      - image: circleci/node:6.15.1-browsers-legacy
+      - image: circleci/node:latest
```

Comment:
You might want to run tests on all versions that you officially support, to avoid accidental incompatible changes.

Reply:
TS config targets ES5, should be safe.